processor/src/operations/ext2_ops.rs

```diff
@@ -0,0 +1,78 @@
+use super::{AdviceProvider, ExecutionError, Felt, Process};
+
+// EXTENSION FIELD OPERATIONS
+// ================================================================================================
+
+impl<A> Process<A>
+where
+    A: AdviceProvider,
+{
+    // ARITHMETIC OPERATIONS
+    // --------------------------------------------------------------------------------------------
+    /// Gets the top four values from the stack [a1, a0, b1, b0], where a = (a1, a0) and
```

Comment:
nit: just to be consistent throughout, let's put `a` deep into the stack and `b` on the top of the stack. So, initial stack state here would look like: `[b1, b0, a1, a0, ...]`.

Reply:
Refactored the method earlier but missed the doc comment.